### DISCUSSION TITLE
Adding a new NFT to an existing collection does not work

### DISCUSSION THREAD

**Discussion by jkdizajn** [kodadot/nft-gallery#277] [General]
Hello,
when I try to add a copy of the NFT to an already created collection, the system will not allow it.
Has anyone encountered this problem?

Link to the video from the preview `https://gateway.pinata.cloud/ipfs/QmQyuKvK9HGYCYPwYQvyxWXXKnZSobcs2JAR91kQ6oJfGk`

**Comment by vikiival** [comment]
Small recap:
@jkdizajn created a single 1-by-1 NFT and his request is possibility to  set `max` attribute  ( create sort of copies :))

Update for #278
